build.sbt

```diff
@@ -6,51 +6,50 @@ description := "Scala GraphQL implementation"
 homepage := Some(url("http://sangria-graphql.org"))
 licenses := Seq("Apache License, ASL Version 2.0" → url("http://www.apache.org/licenses/LICENSE-2.0"))
 
-scalaVersion := "2.12.7"
-crossScalaVersions := Seq("2.11.11", "2.12.7")
+scalaVersion := "2.13.0"
+crossScalaVersions := Seq("2.11.11", "2.12.7", scalaVersion.value)
 
 scalacOptions ++= Seq(
   "-deprecation",
   "-feature",
-  "-Xlint:-missing-interpolator,-unused,_")
+  "-Xlint:-missing-interpolator,_")
 
 scalacOptions ++= {
-  if (scalaVersion.value startsWith "2.12")
-    Seq.empty
-  else
+  if (scalaVersion.value startsWith "2.11")
     Seq("-target:jvm-1.7")
+  else
+    Seq.empty
 }
 
 testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-oF")
 
 libraryDependencies ++= Seq(
   // AST Parser
-  "org.parboiled" %% "parboiled" % "2.1.4",
+  "org.parboiled" %% "parboiled" % "2.1.8",
 
   // AST Visitor
-  "org.sangria-graphql" %% "macro-visit" % "0.1.1",
+  "org.sangria-graphql" %% "macro-visit" % "0.1.2-SNAPSHOT",
 
   // Marshalling
-  "org.sangria-graphql" %% "sangria-marshalling-api" % "1.0.3",
+  "org.sangria-graphql" %% "sangria-marshalling-api" % "2.0.0-SNAPSHOT",
```

Comment:
as I commented in https://github.com/sangria-graphql/sangria-marshalling-api/pull/4#issuecomment-511390492 I think we can make avoid breaking compatibility and use `1.0.4` instead...

Reply:
+1 (for now I've just gone with the versions in the existing PRs for these dependencies).

Reply:
I don't see the harm in breaking binary compatibility in `sangria-marshalling-api` since there is work required in all marshalling glue libraries anyway. Personally I don't like using deprecated stuff when there's a chance to fix it.

Reply:
@ingemaradahl In Scala 2.13 `Traversable` is a type alias for `Iterable`, so even if we keep using it there, that doesn't impose any deprecation cost downstream (users can just treat the return value of `getMapKeys` as an `Iterable` on 2.13).